src/htmlize/patchtml.ml

```diff
@@ -105,6 +105,61 @@ let handle_link ~is_raw link =
                   String.concat "/" (new_link @ target);
     }
 
+open Ez_html
+open Xml
+open Xml_types
+
+(* Function to change static links found in HTML code in md files *)
+let is_edit_tag tag = tag = "a" || tag = "img"
+
+let rec edit_attrs_list ~is_raw attrl =
+  match attrl with
+  | [] -> assert false
+  | (attr_name, attr_value) :: r ->
+    begin
+      match attr_name with
+      | "href" | "src" ->
+        let link =
+          {
+            label = Text([("", "")], "");
+            destination = attr_value;
+            title = None
+          }
+        in
+        let newlink = handle_link ~is_raw link
+        in (attr_name, newlink.destination)::r
+      | _attr -> (attr_name, attr_value)::(edit_attrs_list ~is_raw r)
+    end
+
+let rec markdown_html_patch = function
+  | PCData data -> PCData data
+  | Element (tag, attrs, children) ->
+    match tag with
```

Comment:
I believe there's more, try to find a list of where "hrefs" and "src" can appear. :-)